### PR TITLE
Migrate to LiteDB

### LINKED ISSUE #41: Embedded database support
That's an overhead to use external database for the current tasks. I think we should embed SQLite here.

### PR DESCRIPTION
**Pull request by ForNeVeR**
Closes #41.

### CODE REVIEW

File: EvilPlanner.Backend/Quotes.fs [hunk 1]
```diff
@@ -26,48 +25,53 @@ type Quote (q : Quotation) =
...
+        *> Json.write "source" q.source
 
 let private isoDateFormat = "yyyy-MM-dd"
 
 let private date =
     freya {
         let! value = Freya.getLensPartial <| Route.Atom_ "date"
-        return DateTime.ParseExact (Option.get value, isoDateFormat, CultureInfo.InvariantCulture)
+        return DateTime.ParseExact(
+            Option.get value,
```
Comment (gsomix) [inline EvilPlanner.Backend/Quotes.fs:39]: Where `ArgumentException` will be handled?

File: EvilPlanner.Backend/Quotes.fs [hunk 2]
```diff
@@ -26,48 +25,53 @@ type Quote (q : Quotation) =
...
     freya {
+        let getQuote = fun d -> async { return QuoteLogic.getQuote database d }
         let! date = date
-        let! dbQuote = (Freya.fromAsync QuoteLogic.getQuote) date
+        let! dbQuote = (Freya.fromAsync getQuote) date
         return dbQuote |> Option.map Quote
     } |> Freya.memo
 
-let private checkQuoteByDateExists =
+let private checkQuoteByDateExists cfg =
```
Comment (gsomix) [inline EvilPlanner.Backend/Quotes.fs:53]: `cfg` -> `database`

File: EvilPlanner.Backend/Quotes.fs (hunk 1 above)
Comment (ForNeVeR) [reply EvilPlanner.Backend/Quotes.fs:39]: Well, it's a good idea to add a handler for that, thanks.

File: EvilPlanner.Backend/Quotes.fs (hunk 2 above)
Comment (ForNeVeR) [reply EvilPlanner.Backend/Quotes.fs:53]: Thanks! Fixed.

**Pull request merged**